src/mono/msbuild/apple/build/AppleApp.targets

```diff
@@ -93,6 +93,7 @@
 
     <PropertyGroup>
       <_EnableUnmanagedCallersOnlyMethodsExport Condition="'$(_IsLibraryMode)' == 'true'">true</_EnableUnmanagedCallersOnlyMethodsExport>
+      <_UseAotDataFile Condition="'$(_IsLibraryMode)' == 'true' and '_UseAotDataFile' == ''">false</_UseAotDataFile>
```

Comment:
Shouldn't the apple targets have same features as the Android? Missing the callback compiler arguments needed for library mode.

Reply:
Good catch. I didn't focus on fleshing out the apple side since we haven't yet finished getting the LibraryBuilder working for the Apple side anyways.
Would it be better to just add the Apple side of things when we get things rolling there (and do the same for wasm/wasi when we start on that front + other platforms)?
I'll go ahead and add all the pieces for Apple

Reply:
Yes, add at least the platform we know users will start out testing.